### PR TITLE
EPMRPP-37349: Administration/Projects - delete projects action

### PR DESCRIPTION
**Pull request by margalencia**

### CODE REVIEW

File: app/src/pages/admin/projectsPage/projectsToolbar/projectsToolbar.jsx [hunk 1]
```diff
@@ -45,18 +96,89 @@ export class ProjectsToolbar extends Component {
...
   static defaultProps = {
     viewMode: GRID_VIEW,
+    selectedProjects: [],
   };
 
   onExportProjects = () => {
     window.location.href = URLS.exportProjects();
   };
 
+  get selectedProjectsNames() {
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/projectsToolbar/projectsToolbar.jsx]: getSelectedProjectNames

Comment (margalencia) [reply app/src/pages/admin/projectsPage/projectsToolbar/projectsToolbar.jsx]: `get getSelectedProjectNames`?

Comment (DiscoElevator) [reply app/src/pages/admin/projectsPage/projectsToolbar/projectsToolbar.jsx]: just `getSelectedProjectNames` without getter
https://github.com/reportportal/service-ui/pull/1227#discussion_r264651260

Comment (margalencia) [reply app/src/pages/admin/projectsPage/projectsToolbar/projectsToolbar.jsx]: done

**Pull request merged**